src/Manager.php

```diff
@@ -164,10 +163,16 @@ public function parseIncludes($includes): self
                 $modifierArr[$modifierName] = explode($this->paramDelimiter, $modifierParamStr);
             }
 
-            $this->includeParams[$includeName] = $modifierArr;
+            if (!isset($this->includeParams[$includeName])) {
+                $this->includeParams[$includeName] = $modifierArr;
```

Comment:
did you consider collecting all values into a list of `modifierArr`?

Reply:
Yeah, but I'm not sure if this is the best design choice. I think it would have to be discussed with upstream. I wanted to limit changes to a bugfix and keep backward compatibility.